extensions/functions_arithmetic.yaml

```diff
@@ -756,6 +756,8 @@ aggregate_functions:
         options:
           overflow:
             values: [ SILENT, SATURATE, ERROR ]
+          skip_nulls:
```

Comment:
I'm wondering if we should add this property to the aggregate function invocation as opposed to every aggregate function. It feels kind of like aggregate input sorting or some other function-impl independent concept. Thoughts @westonpace @gforsyth ?

Reply:
I agree that if we're specifying an option for every function definition that we should move it up to the invocation

Reply:
That makes sense, will close this PR and have opened https://github.com/substrait-io/substrait/issues/401 instead.